Tribler/Test/Core/test_sqlitecachedb.py

```diff
@@ -35,13 +35,20 @@ def tearDown(self):
         self.sqlite_test = None
 
     @blocking_call_on_reactor_thread
-    def test_create_db(self):
+    def test_create_person_db(self):
```

Comment:
There are now two databases to test with, one with a table only containing one column (cars) and the other one with person so there are two columns (firstname and lastname).